Example/Podfile

```diff
@@ -1,6 +0,0 @@
-source 'https://github.com/CocoaPods/Specs.git'
-use_frameworks!
-
-target 'YelpAPI_Example', :exclusive => true do
```

Comment:
Hm I still see this as a target, but there doesn't appear to be any schemes that builds this target by default.